domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -287,6 +290,20 @@ constructor(
                 var prevTransientSettings = initialTransientSettings
                 cameraProvider.runWith(sessionSettings.cameraSelector, useCaseGroup) { camera ->
                     Log.d(TAG, "Camera session started")
+
+                    launch {
+                        cameraEvents.consumeAsFlow().collect {
```

Comment:
This is probably OK as is for now, but I would recommend wrapping this in a `getSurfaceRequests().filterNotNull().map<MeteringPointFactory> { ... }.collectLatest { }` rather than creating a new `MeteringPointFactory` for every event.  This allows you to make `tapToFocus` no longer need to be a `suspend fun` (though we may need to do that later anyways for a response).

You would also want to break out the `FocusMeteringEvent` into its own channel so you can have a conflated channel that only delivers the latest event. 

I'm fine with whichever way you want to go.